.github/workflows/dependent-issues.yml

```diff
@@ -0,0 +1,36 @@
+name: Dependent Issues
+
+on:
+  issues:
```

Comment:
`issues` includes pull requests, right? Since we disable `check_issues` below, is it enough to just have `pull_request_target`? Or alternatively do we need `pull_request_target` if `issues` includes PRs? I'm not clear on the details here. Do we need to watch issues in case a PR depends on an issue?

Reply:
Yeah, we have to watch issues in case a PR depends on an issue.

Reply:
This part of the workflow file is just copied from the example in the [dependent-issues README](https://github.com/z0al/dependent-issues#usage), so I'm trusting that these triggers are the correct / minimal ones.